src/sourceViewObserver.tsx

```diff
@@ -0,0 +1,197 @@
+        ];
+        const suffixObservedClassName = [
+            '',
+            ' cm-list-1', ' cm-list-2', ' cm-list-3', 
+            ' cm-quote cm-quote-1', ' cm-strong'
+        ];
+        const baseTargetClassName = 'cm-hmd-internal-link cm-link-alias';
+        for (const className of prefixObservedClassName) {
+            this.targetClassNameSet.add(className + baseTargetClassName);
+        }
+        for (const className of suffixObservedClassName) {
+            this.targetClassNameSet.add(baseTargetClassName + className);
+        }
+    }
+
+    initObserver() {
+        const activeLeaf = this.plugin.app.workspace.getActiveViewOfType(MarkdownView);
+        if (!activeLeaf) return;
+    
+        const that = this;
+        const observedClassName = 'cm-hmd-internal-link cm-link-has-alias';
+        const targetClassName = 'cm-hmd-internal-link cm-link-alias';
+        const filePath = this.plugin.app.workspace.getActiveFile().path;
+        const observeCallback = function(mutations: MutationRecord[]) {
+            that.plugin.log('-----------------------------');
+            for (const mutation of mutations) {
+                if (mutation.type == 'childList') {
+                    for (const addedNode of mutation.addedNodes) {
+                        const addedElement = addedNode as Element;
+                        if (addedElement.className && addedElement.className.indexOf(observedClassName) >= 0) {
+                            that.linkOnFocus(addedElement);
+                        }
+                    }
+                    for (const removedNode of mutation.removedNodes) {
+                        const removedElement = removedNode as Element;
+                        if (removedElement.className && removedElement.className.indexOf(observedClassName) >= 0) {
+                            const removedLinkText = removedElement.textContent;
+                            that.plugin.log('remove');
+                            that.linkOnBlur(mutation.target as Element, removedLinkText, filePath, targetClassName);
+                        }
+                    }
+                } else if (mutation.type == 'attributes') {
+                    if (mutation.oldValue && mutation.oldValue.indexOf(observedClassName) >= 0) {
+                        that.plugin.log('attri');
+                        that.linkOnBlur(mutation.target.parentNode as Element, '', filePath, targetClassName);
+                    }
+                }
+            }
+        }
+        this.observer = new MutationObserver(observeCallback);
+    }
+
+    getObserver(): MutationObserver {
+        return this.observer;
+    }
+
+    setObserver(observer: MutationObserver) {
+        this.observer = observer;
+    }
+
+    watch() {
+        const activeLeaf = this.plugin.app.workspace.getActiveViewOfType(MarkdownView);
+        if (!activeLeaf) return;
+    
+        const regex2Find = /(?<=[\[]{2})[^\]]*(?=[\]]{2})/gm;
```

Comment:
Fun fact. Plugin didn't start on iOS(#284) because of this line🤡 

Turned out it because Safari doesn't support `?<=` and throws syntax error. I didn't find other way to exclude `[[` with regex and added removing it with `hrefLink.substring(2)`.